api/internal.c

```diff
@@ -105,7 +105,11 @@ void _function_callback_wrapper(mambo_context *ctx, watched_func_t *func) {
   ctx->code.func_name = func->name;
 
   if (func->post_callback != NULL) {
-    emit_push(ctx, (1 << es) | (1 << lr));
+    #if defined __arm__ || __aarch64__
```

Comment:
No need for this change. See the comments above for adding an LR alias and we also need to define `es` on RISC-V.